ambry-clustermap/src/test/java/com.github.ambry.clustermap/HelixClusterManagerTest.java

```diff
@@ -428,12 +493,46 @@ public void sealedReplicaChangeTest() {
     assertStateEquivalency();
   }
 
+  /**
+   * Test that ClusterManger will use seal state in PartitionOverride when {@link ClusterMapConfig#clusterMapEnablePartitionOverride}
+   * is enabled. This also tests that InstanceConfig changes won't affect any seal state of partition if clusterMapEnablePartitionOverride
+   * is enabled.
+   */
+  @Test
+  public void clusterMapOverrideEnabledTest() throws Exception {
+    if (overrideEnabled) {
```

Comment:
is there a test that makes sure overrides are not used (even if they exist) if the config is not enabled?